Examples/UIExplorer/PanResponderExample.js

```diff
@@ -78,13 +80,17 @@ var PanResponderExample = React.createClass({
 
   _highlight: function() {
     this.circle && this.circle.setNativeProps({
-      backgroundColor: processColor(CIRCLE_HIGHLIGHT_COLOR)
+      style: {
+        backgroundColor: processColor(CIRCLE_HIGHLIGHT_COLOR)
+      }
     });
   },
 
   _unHighlight: function() {
     this.circle && this.circle.setNativeProps({
-      backgroundColor: processColor(CIRCLE_COLOR)
+      style:{
```

Comment:
Ty, I had tired to run the linter (`npm run lint`) but there were like 50 failures in other files. Is there a better way I can lint? I wasn't sure how to go about the suggested method in [CONTRIBUTING.md](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#pull-requests)